docs/application-connector/docs/08-04-register-secured-api.md

```diff
@@ -104,3 +106,29 @@ kubectl -n kyma-integration patch secrets app-{APP_NAME}-{SERVICE_ID} --patch 'd
   crt: {BASE64_ENCODED_CRT}
   key: {BASE64_ENCODED_KEY}'
 ```
+
+## Register a CSRF-protected API
+
+The application-registry supports CSRF tokens as an additional layer of API protection. To register a CSRF-protected API, you must add the `credentials.{AUTHENTICATION_METHOD}.csrfInfo` object to the `api` section of the service registration request body.
+
+Include this field in the service registration request body:
+
+| Field | Description |
+|:-----:|:-----------:|
+| **tokenEndpointURL** | The URL to the upstream service endpoint that exposes CSRF tokens. |
+
+This is an example of the `api` section of the request body for an API secured with both Basic Authentication and a CSRF token.
```

Comment:
I guess the CSFR token verification works as an ADDITIONAL security measure for basic auth- or OAUTH- secured APIs? If so, please make this clear in the introductory paragraph.

Reply:
Nevermind